source/documentation/tools.md

```diff
@@ -185,6 +185,16 @@ It has some significant downsides. It can be quite temperamental, and difficult
 
 Furthermore, the Analytical Platform version of RStudio runs on a Linux virtual machine, and CRAN mirrors do not provide Linux compiled binaries for packages. This means that packages need to be compiled on the Analytical Platform every time they're installed, which can take a long time. This means a long wait when doing `install.packages` both in an RStudio session, and when running a Docker build for an RShiny application.
 
+### Packrat usage
```

Comment:
Maybe this could be changed to just 'Usage' given it sits within the 'Packrat' section.
```suggestion
### Packrat usage
```